CheckDrive.Mobile/CheckDrive.Mobile/CheckDrive.Mobile.Android/Resources/values/styles.xml

```diff
@@ -1,18 +1,23 @@
 <?xml version="1.0" encoding="utf-8" ?>
 <resources>
 
-  <style name="MainTheme" parent="MainTheme.Base">
-    <!-- As of Xamarin.Forms 4.6 the theme has moved into the Forms binary -->
-    <!-- If you want to override anything you can do that here. -->
-    <!-- Underneath are a couple of entries to get you started. -->
+	<!-- Main Theme used by the entire application -->
+	<style name="MainTheme" parent="MainTheme.Base">
+		<!-- Customize your app's color scheme -->
+		<item name="colorPrimaryDark">#1C375C</item>
+		<item name="colorAccent">#1C375C</item>
+		<item name="android:datePickerDialogTheme">@style/CustomDatePickerDialog</item>
+	</style>
 
-    <!-- Set theme colors from https://aka.ms/material-colors -->
-    <!-- colorPrimary is used for the default action bar background -->
-    <!--<item name="colorPrimary">#2196F3</item>-->
-    <!-- colorPrimaryDark is used for the status bar -->
-    <item name="colorPrimaryDark">#1C375C</item>
-    <!-- colorAccent is used as the default value for colorControlActivated
-         which is used to tint widgets -->
-    <item name="colorAccent">#1C375C</item>
-  </style>
-</resources>
\ No newline at end of file
+	<!-- Base application theme -->
+	<style name="MainTheme.Base" parent="Theme.AppCompat.Light.DarkActionBar">
+		<!-- Add any other global styles here -->
+	</style>
+
+	<!-- Custom style for DatePickerDialog -->
+	<style name="CustomDatePickerDialog" parent="ThemeOverlay.MaterialComponents.MaterialAlertDialog">
+		<item name="colorAccent">#1C375C</item>
+		<item name="android:textColorPrimary">#1C375C</item>
+	</style>
```

Comment:
Have you tested the application? Are you sure this won't break any other places?